comit/src/btsieve/bitcoin/bitcoind_connector.rs

```diff
@@ -2,7 +2,7 @@ use crate::{
     btsieve::{BlockByHash, LatestBlock},
     ledger,
 };
-use anyhow::Context;
+use anyhow::{Context, Result};
```

Comment:
I would rather have `pub use anyhow::Result` in the `lib.rs` to make it obvious that we use `anyhow::Error` across the crate.

Then import with `crate::Result` everywhere.

Reply:
hum, interesting idea but not sure if it is actually better. Creates another layer of direction to resolve imports no?

Reply:
🤷🏻‍♂️
Let's wait for someone else to build on top of the comit crate and tell us :D